src/euphorie/content/browser/country.py

```diff
@@ -101,3 +104,38 @@ class Tools(CountryView):
     def get_tools(self, sector_id):
         sector = self.context.get(sector_id)
         return getSurveys(sector)
+
+    def download_csv(self):
+        fieldnames = [
+            "Sector",
+            "Tool Title",
+            "Version Title",
+            "URL",
+            "published",
+            "obsolete",
+        ]
+        buffer = StringIO()
+        writer = csv.DictWriter(buffer, fieldnames=fieldnames, dialect="bilbomatica")
+        writer.writeheader()
+        for sector in self.sectors:
+            for group in self.get_tools(sector["id"]):
+                for survey in group["surveys"]:
+                    writer.writerow(
+                        {
+                            "Sector": sector["title"],
+                            "Tool Title": group["title"],
+                            "Version Title": survey["title"],
+                            "URL": survey["url"],
+                            "published": "published" if survey["published"] else "",
+                            "obsolete": "obsolete" if group["obsolete"] else "",
+                        }
+                    )
+        csv_data = buffer.getvalue()
+        buffer.close()
+        response = self.request.RESPONSE
+        response.setHeader(
+            "Content-Disposition",
+            f"attachment; filename=oira_tools_{self.context.id}.csv",
```

Comment:
just a matter of taste, should we include the date isostring also in the filename? I personally would do so.